extra-scientific/bedtools/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=bedtools
+PKGSEC=science
+PKGDES="A powerful tool for genome arithmetic"
+export prefix=/usr
```

Comment:
Tested on ACBS that the prefix was not defined. Maybe due to the differences in capitalization? From the makefile, it takes prefix but not PREFIX.